Variables.pyx

```diff
@@ -295,28 +295,26 @@ cdef class GridMeanVariables:
             Stats.write_profile('HQTcov_mean',self.HQTcov.values[self.Gr.gw:self.Gr.nzg-self.Gr.gw])
 
         Stats.write_profile('cloud_fraction_mean',self.cloud_fraction.values[self.Gr.gw:self.Gr.nzg-self.Gr.gw])
+        Stats.write_ts('cloud_cover_mean', self.cloud_cover)
```

Comment:
Just making sure we have this right: The new cloud_cover from Turbulence_prognosticTKE.pyx corresponds to a minimum overlap of clouds. The previous version was maximum overlap assumption.

Reply:
understood.